src/renderer/prefComponents/imageUploader/index.vue

```diff
@@ -1,19 +1,28 @@
 <template>
   <div class="pref-image-uploader">
     <h4>Image Uploader</h4>
-    <section class="current-uploader">
-      <div>The current image uploader is <span class="uploader">{{currentUploader}}</span>.</div>
-    </section>
-    <separator></separator>
     <section class="configration">
       <el-tabs v-model="activeTab">
+        <el-tab-pane label="Overview" name="none">
```

Comment:
I prefer not to put `Overview` as one tab of image uploaders, because it's a image uploader, I prefer put it above the tabs?